documentation/docs/adapters/RPGRegionsAdapter/entries/action/discover_region.mdx

```diff
@@ -0,0 +1,17 @@
+import {ActionsField, EntryField} from "@site/src/components/EntryField";
+
+# Discover Region Action
+
+The `Discover Region Action` is used to add a discovery into a user's account.
+
+## How could this be used?
+
+This action could be used to reward the player for completing a task/quest.
```

Comment:
This is already okay but it could be more descriptive. Maybe include a specific scenario.